### DISCUSSION TITLE
How to access from gcp

### DISCUSSION THREAD

**Discussion by iamtvk** [coder/code-server#4559] [Q&A]
How to access code-server if i am using GCP ? 
i did created a "allow-8080"  but i can't access it

**Comment by lg-nineoneninethreenine** [comment]
Have you tried [this](https://stackoverflow.com/questions/59093373/error-could-not-connect-to-cloud-shell-on-port-8080-while-connecting-the-vis)?

**Reply by jsjoeio** [reply]
^+1 to that. There are also some docs [here](https://coder.com/docs/code-server/latest/requirements#set-up-a-vm-on-google-cloud) (though they don't focus as much on the exposure part). You could also run `localtunnel` or `ngrok` on your GCP VM and expose it that way.
